docs/UserGuide.md

```diff
@@ -63,19 +81,20 @@ before continuing to type!
 
-Come up with a new innovative recipe and want to store it for future reference,
-and want to classify it by ingredients? Simply run the `add` command, and follow the prompts!
+Come up with a new innovative recipe and afraid that you'll forget about it in the future,
+or just want to share it with your friends? Simply run the `add` command, and follow the prompts to save different
+information about your recipe such as ingredients and steps! 
 
 Add Format:
 
-| Fields         | Format, Examples                                          
-|----------------|------------------------------------------------------|
-| **Name**       | `n/RECIPE_NAME` <br/> e.g., `n/Caesar Salad`     |
-| **Duration**   | `d/RECIPE_DURATION` <br/> e.g., `d/15 minutes`                                                               |
-| **Portion**    | `p/RECIPE_PORTION` <br/> e.g., `p/1-2 servings`  |
-| **Ingredients**| `i/-n INGREDIENT_NAME [-a INGREDIENT_AMOUNT] [-e ESTIMATED AMOUNT] [-cn COMMON NAME] [-r REMARKS]...` <br/> e.g.,` i/-a 2 tbsp -n Cider Vinegar -cn Vinegar -r Optional` <br/> e.g., ` i/-n Pasta`|
-| **Steps** | `s/RECIPE_STEPS`<br/> e.g., `s/Add cider vinegar to salad`|
-| **Tags**    | `t/RECIPE_TAGS` <br/> e.g. `t/Vegetarian`         |
+| Fields          | Format, Examples                                                                                                                                                                                   |    
+|-----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Name**        | `n/RECIPE_NAME` <br/> e.g., `n/Caesar Salad`                                                                                                                                                       |
+| **Duration**    | `d/RECIPE_DURATION` <br/> e.g., `d/15 minutes`                                                                                                                                                     |
+| **Portion**     | `p/RECIPE_PORTION` <br/> e.g., `p/1-2 servings`                                                                                                                                                    |
+| **Ingredients** | `i/-n INGREDIENT_NAME [-a INGREDIENT_AMOUNT] [-e ESTIMATED AMOUNT] [-cn COMMON NAME] [-r REMARKS]...` <br/> e.g.,` i/-a 2 tbsp -n Cider Vinegar -cn Vinegar -r Optional` <br/> e.g., ` i/-n Pasta` |
+| **Steps**       | `s/RECIPE_STEPS`<br/> e.g., `s/Add cider vinegar to salad`                                                                                                                                         |
+| **Tags**        | `t/RECIPE_TAGS` <br/> e.g. `t/Vegetarian`                                                                                                                                                          |
```

Comment:
i already updated this table (and moved it to a different section) in the `v1.4-documentation` branch, you may want to pull and merge that instead